app/javascript/gobierto_dashboards/webapp/components/SearchFilter.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="gobierto_dashboards-search-container">
+    <label class="gobierto_dashboards-search-container-label">
+      {{ labelSearch }}:
+    </label>
+    <div class="gobierto_dashboards-search-container-wrapper-input">
+      <div class="search-box">
+        <i class="fas fa-search gobierto_dashboards-search-btn-search" />
+        <input
+          v-model="search"
+          type="text"
+          :placeholder="labelPlaceholder"
+          class="gobierto_dashboards-search-container-input"
+          @input="handlerFilterItems"
+        >
+        <i
```

Comment:
Could you hide this when the value of the input is nil?

Reply:
Done!
![alianza_input](https://user-images.githubusercontent.com/2649175/88773816-ebc91900-d182-11ea-80b8-a6248e29b036.gif)